modules/common/src/test/java/com/ibm/cloud/cloudant/internal/ValidationTest.java

```diff
@@ -14,9 +14,14 @@
 
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.concurrent.TimeUnit;
```

Comment:
Remove left over imports

Reply:
yes, please clean up these unnecessary imports

Reply:
Removed in 4b0c8649